packages/client/src/stores/notification.js

```diff
@@ -60,7 +61,7 @@ const createNotificationStore = () => {
     if (autoDismiss) {
       setTimeout(() => {
         dismiss(_id)
-      }, NOTIFICATION_TIMEOUT)
+      }, duration)
```

Comment:
Since this is the only place we actually use `duration`, I think the best option is to ensure that we have a valid value right here. I would update this to `duration || DEFAULT_NOTIFICATION_TIMEOUT` as that account for `undefined`, `null`, empty string and even 0. It's a more robust approach and just defaulting it in the function signature 👍